src/base/CharacterSet.cc

```diff
@@ -168,3 +168,18 @@ CharacterSet::RFC3986_UNRESERVED()
     return *chars;
 }
 
+const CharacterSet &
+CharacterSet::libcSpace()
+{
+    const auto creator = []() -> auto {
+        const auto chars = new CharacterSet("libcSpace");
+        for (size_t idx = 0; idx < 256; ++idx) {
```

Comment:
We could std::numeric_limits<unsigned char>::max() instead of the numeric constant.

Reply:
Done in 4a565ed3. Technically, there were _two_ explicit numeric constants, so I converted both.